source/basic.tex

```diff
@@ -5168,12 +5167,16 @@
 \end{example}
 
 \pnum
+\indextext{array!\idxcode{const}}%
+An array type whose elements are cv-qualified
+is also considered to have the same cv-qualifications
+as its elements.
+\begin{note}
 Cv-qualifiers applied to an array
 type attach to the underlying element type, so the notation
 ``\cv{}~\tcode{T}'', where \tcode{T} is an array type, refers to
-an array whose elements are so-qualified. An array type whose elements
-are cv-qualified is also considered to have the same cv-qualifications
-as its elements.
+an array whose elements are so-qualified.
+\end{note}
```

Comment:
Let's reorder this to right after the "from which it is compounded" rule, so the reader isn't left wondering what happens for array types for several paragraphs while we talk about other things.

Reply:
@zygoloid Good idea. Both changes applied.